docs/developer/architecture/volume.rst

```diff
@@ -157,6 +148,28 @@ Similarly, our **Volume** object will have the following states:
 * **Terminating**: cleanup of the backing storage in progress (e.g.
   an asynchronous Salt call is still running).
 
+Persistent block device naming
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+In order to have a reliable automount through kubelet, we need to create the
+underlying **PersistentVolume** using a persistent name for the backing storage
+device. We use different strategies according to the **Volume** type:
+
+* **sparseLoopDevice** and **rawBlockDevice** with a filesystem: during the
+  formatting, we set the filesystem UUID to the **Volume** UUID and use
+  ``dev/disk/by-uuid/<volume-uuid>`` as device path.
+* **sparseLoopDevice** without filesystem: we create a GUID Partition Table on
+  the sparse file and create a single partition encompassing the whole device,
+  setting the GUID of the partition to the **Volume** UUID. We can then use
```

Comment:
Should be `the UUID of the partition`, no ? Or maybe I'm lost with all these UUID/GUID. 
```suggestion
  setting the UUID of the partition to the **Volume** UUID. We can then use
```